docs/docs/how-tos/graph-api.md

```diff
@@ -2485,9 +2485,10 @@ Recursion Error
 ```
 :::
 
-??? example "Extended example: return state on hitting recursion limit"
 
     :::python
```

Comment:
I move these to start tags above the callouts to remove them from the JS versions. I don't know if we're supposed to have equivalent callouts or not for the JS version, but I made this change initially to just hide the dropdowns from the JS version.